tests/pkg_libbase58/main.c

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2019 Kaspar Schleiser <kaspar@schleiser.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup    tests
+ * @{
+ *
+ * @file
+ * @brief      Tests for pkg libbase58
+ *
+ * @author     Kaspar Schleiser <kaspar@schleiser.de>
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdint.h>
+
+#include "libbase58.h"
+#include "embUnit.h"
+
+static void setUp(void)
+{
+}
```

Comment:
You can remove that if it is empty (and NULL instead in the test caller generation).

Reply:
Ping?

Reply:
fixed!